src/hooks/privateChats.ts

```diff
@@ -165,12 +165,12 @@ export const useRecipientChat = (recipientId: string) => {
   const recipient = withId(worldUsersById[recipientId], recipientId);
 
   const sendMessageToSelectedRecipient: SendMesssage = useCallback(
-    (text: string) => {
+    (props) => {
       if (!userId) return;
 
       const message = buildMessage<PrivateChatMessage>({
         from: userId,
-        text,
+        ...props,
         to: recipientId,
       });
```

Comment:
can we explicitly destruct `props` here? `({ text, isQuestion }) => { ... }` to understand what is the structure of that parameter without going deeper into types?

Reply:
Is props explicitly typed anymore? We might be getting it via SendMessage type, hard to tell without looking deeper at the code.

How many args are we using from it? If only a couple, then agreed with @sunny-viktoryia

Reply:
https://github.com/sparkletown/sparkle/pull/1424/commits/25fa94132d1397282d910908ab45e4725ac09d4e